packages/expo-dev-menu/ios/DevMenuRCTBridge.mm

```diff
@@ -136,4 +137,21 @@ - (RCTBridge *)createBridgeWithDelegate:(id<RCTBridgeDelegate>)delegate launchOp
   return [[DevMenuRCTBridge alloc] initWithDelegate:delegate launchOptions:launchOptions];
 }
 
+- (NSArray<id<RCTBridgeModule>> *)extraModulesForBridge:(RCTBridge *)bridge
+{
+  NSMutableArray<id<RCTBridgeModule>> *modules = [NSMutableArray new];
+  [modules addObject:[[DevClientNoOpLoadingView alloc] init]];
+  [modules addObject:[[DevMenuRCTDevSettings alloc] init]];
+
+  return modules;
+}
+
+- (Class)getModuleClassFromName:(const char *)name {
+  // Overrides DevLoadingView ("Connect to Metro to develop JavaScript") as no-op for dev menu
+  if (strcmp(name, "DevLoadingView") == 0) {
+    return [DevClientNoOpLoadingView class];
+  }
+  return [super getModuleClassFromName:name];
+}
+
```

Comment:
move from original `EXDevClientReactNativeFactory`. the correct place to setup these seems to be a RCTReactNativeFactoryDelegate, not factory itself.